manifests/credentials.pp

```diff
@@ -12,10 +12,7 @@
 # See the License for the specific language governing permissions and
```

Comment:
should we keep this text or maybe move it to `@summary`?

Reply:
I wasn't quite sure. Perhaps we can just drop it altogether since the whole module is already licensed.